01-node-tutorial/answers/practice3.js

```diff
@@ -0,0 +1,16 @@
+fs = require('fs').promises
+
+const makeFile = async () => {
+    try{
+        
+        await fs.writeFile('./content/practice2.txt',`This is the first line. \n`)
+        for(i=2; i<11; i++){
```

Comment:
```suggestion
        for(let i=2; i<11; i++){
```
This works fine as is but it's a good idea to always declare your variables so it's clear that the intent is to create a new variable rather than reuse an existing one.